src/components/NewMessageModal.vue

```diff
@@ -17,8 +17,7 @@
 			:draft="saveDraft"
 			:send="sendMessage"
 			:forwarded-messages="forwardedMessages"
-			@discard-draft="discardDraft"
-			@close="$emit('close')" />
+			@discard-draft="discardDraft" />
```

Comment:
This change is not related but Composer never emits `close`.

## Proof
```js
$ grep -n '\$emit' src/components/Composer.vue
872:                            this.bus.$emit('insertSignature', signatureValue, this.selectedAlias.signatureAboveQuote)
904:                    this.bus.$emit('onAddLocalAttachment')
908:                    this.bus.$emit('onAddCloudAttachment')
912:                    this.bus.$emit('onAddCloudAttachmentLink')
1055:                   this.$emit('discard-draft', id)
```

Reply:
Could we rename a outbox mutation "stopMessage" to something like "deleteScheduledValue" ? it will be a bit clearer

Reply:
I'm the original author of the name so take this with caution but I still think the name makes sense. The action name describes *what* it does, not *how*.

We also have *sendMessage* and not *postMessageToServer* and similar. It's a always a bit abstract.

*deleteScheduledValue* is too specific to the implementation IMO.